lib/bibliothecary/parsers/haskell.rb

```diff
@@ -0,0 +1,32 @@
+require "json"
+
+module Bibliothecary
+  module Parsers
+    class Haskell
```

Comment:
The naming convention that I've been following has been to name the class the same as the registry name in Libraries.io, which in this case is `Hackage`

Reply:
Also wasn't clear.  Thanks again for correcting.